glue/plugins/wcs_autolinking/wcs_autolinking.py

```diff
@@ -12,21 +14,45 @@ class IncompatibleWCS(Exception):
 
 def get_cids_and_functions(wcs1, wcs2, pixel_cids1, pixel_cids2):
 
-    def forwards(*pixel_input):
-        return pixel_to_pixel(wcs1, wcs2, *pixel_input)
+    if len(pixel_cids1) != 1:
+        def forwards(*pixel_input):
+            return pixel_to_pixel(wcs1, wcs2, *pixel_input)
 
-    def backwards(*pixel_input):
-        return pixel_to_pixel(wcs2, wcs1, *pixel_input)
+        def backwards(*pixel_input):
+            return pixel_to_pixel(wcs2, wcs1, *pixel_input)
 
-    pixel_input = (0,) * len(pixel_cids1)
+        pixel_input = (0,) * len(pixel_cids1)
 
-    try:
-        forwards(*pixel_input)
-        backwards(*pixel_input)
-    except Exception:
-        return None, None, None, None
+        if len(pixel_cids1) == 1:
+            pixel_input = np.zeros((1,1))
```

Comment:
Can this ever be satisfied since we are in the != 1 case?

Reply:
Sorry you are absolutely right, I was copying and pasting things and forgot to edit this out.